java-checks/src/main/java/org/sonar/java/checks/maven/helpers/MavenDependencyCollector.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * SonarQube Java
+ * Copyright (C) 2012-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.java.checks.maven.helpers;
+
+import com.google.common.collect.ImmutableList;
+import org.sonar.maven.model.maven2.Build;
+import org.sonar.maven.model.maven2.BuildBase;
+import org.sonar.maven.model.maven2.Dependency;
+import org.sonar.maven.model.maven2.DependencyManagement;
+import org.sonar.maven.model.maven2.DependencyManagement.Dependencies;
+import org.sonar.maven.model.maven2.MavenProject;
+import org.sonar.maven.model.maven2.MavenProject.Profiles;
+import org.sonar.maven.model.maven2.Plugin;
+import org.sonar.maven.model.maven2.Profile;
+
+import javax.annotation.Nullable;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class MavenDependencyCollector {
+
+  private static final List<Dependency> EMPTY_LIST = ImmutableList.of();
```

Comment:
@m-g-sonar you don't need guava, you can use: `java.util.Collections.emptyList()`.